### PR TITLE
[WIP] Add `stats_sender_id` in `ClientAlgoExecutor`

### PR DESCRIPTION
**Pull request by KumoLiu**
Fixes # .

### Description
Part of https://github.com/Project-MONAI/MONAI/pull/6220 for `FLStatsHander`

### Types of changes

- [x] Non-breaking change (fix or new feature that would not break existing functionality).
- [ ] Breaking change (fix or new feature that would cause existing functionality to change).
- [ ] New tests added to cover the changes.
- [ ] Quick tests passed locally by running `./runtest.sh`.
- [ ] In-line docstrings updated.
- [ ] Documentation updated.

### CODE REVIEW

File: integration/monai/monai_nvflare/client_algo_executor.py [hunk 1]
```diff
@@ -56,12 +57,14 @@ def __init__(
 
         Args:
             client_algo_id (str): id pointing to the client_algo object
+            stats_sender_id (str): id pointing to the AnalyticsSender object
```
Comment (YuanTingHsieh) [inline integration/monai/monai_nvflare/client_algo_executor.py:60]: should it be `(str, optional)`?

Comment (KumoLiu) [reply integration/monai/monai_nvflare/client_algo_executor.py]: Thanks! Have updated.

**Comment by KumoLiu** [comment]
Update: Waiting for @chesterxgchen and @holgerroth to discuss a proposal.

**Comment by YuanTingHsieh** [comment]
The changes in this PR has been included in https://github.com/NVIDIA/NVFlare/pull/1925.
Let's discuss next steps in follow up meetings.

**Pull request closed without merging**